conan/cli/commands/graph.py

```diff
@@ -83,6 +85,37 @@ def graph_build_order(conan_api, parser, subparser, *args):
     return install_order_serialized
 
 
+@conan_subcommand(formatters={"text": cli_out_write})
+def graph_build_order_prepare(conan_api, parser, subparser, *args):
```

Comment:
I think that maybe the name of the subcommand it's not so intuitive, what about something like:
- conan graph install-prepare
- conan graph install-config
Maybe more explicit like this?
- conan graph generate-install-args
-